samples/storageclass/storageclass-xfs.yaml

```diff
@@ -33,7 +33,7 @@ parameters:
   # Filesytem type for volumes created by storageclass
   # Allowed values: "ext4" or "xfs"
```

Comment:
Should the allowed values also include the default of "None" with an explanation of behavior if not set? Thanks!

Reply:
Will follow up on this and get back to you

Reply:
Hey Aron! Did some looking into it in standup and it appears that ext4 is actually the default value, so I'll close this PR and comment in the story that triggered it explaining why PFlex is not making this change.